### ISSUE TITLE
Only send URL in crash reports after affirmative action

### ISSUE THREAD

**Issue by snorp** [mozilla-mobile/android-components#7248]
We appear to be sending the URL along with crashes without asking users if this is ok.

**Comment by pocmo** [comment]
What? Really? @rocketsroger Did you add anything like that?

**Comment by pocmo** [comment]
Looks like the GV implementation of `CrashReporter` has a list of keys it ignores:
https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/CrashReporter.java#51-54

It looks like our implementation does not? If so then this may have been happening since we switched to our own uploader?

**Comment by rocketsroger** [comment]
Yeah my implementation did not ignore some of the keys that GV implementation ignores.  I'll fix this today.

**Comment by snorp** [comment]
Filtering out by default is good, but I think we'd also like to be able to send the URL if the user consents. Could we add a checkbox somehwere?

**Comment by pocmo** [comment]
We have some issues about those other fields:
* https://github.com/mozilla-mobile/android-components/issues/3260
* https://github.com/mozilla-mobile/android-components/issues/3261
* https://github.com/mozilla-mobile/android-components/issues/3262
